README.md

```diff
@@ -34,6 +34,7 @@ It's symfony website-skeleton project with some additional tools to validate cod
 
 1. Run `symfony server:start` to launch your local php web server
 2. Run `yarn run dev --watch` to launch your local server for assets
+3. configurer le fichier (.env) pour pouvoir recevoir un email.
```

Comment:
You must at least write the name of the environnement variable to add.
How do you expect someone to get them ? by reading all the code before install ?